addons/stock_account/wizard/stock_picking_return.py

```diff
@@ -17,11 +17,14 @@ def default_get(self, default_fields):
     def _create_returns(self):
         new_picking_id, pick_type_id = super(StockReturnPicking, self)._create_returns()
         new_picking = self.env['stock.picking'].browse([new_picking_id])
+        self._mark_return_moves(new_picking)
+        return new_picking_id, pick_type_id
+        
+    def _mark_return_moves(self, new_picking):
         for move in new_picking.move_lines:
-            return_picking_line = self.product_return_moves.filtered(lambda r: r.move_id == move.origin_returned_move_id)
+            return_picking_line = self.product_return_moves.filtered(lambda r: r.move_id == move.origin_returned_move_id)[0]
```

Comment:
```suggestion
            return_picking_line = self.product_return_moves.filtered(lambda r: r.move_id == move.origin_returned_move_id)[:1]
```
this way avoid error if no moves